src/ambianic/pipeline/ai/fall_detect.py

```diff
@@ -35,6 +35,7 @@ def __init__(self,
 
         self._pose_engine = PoseEngine(self._tfengine)
         self._fall_factor = 60
+        self.pose_confidence_threshold = kwargs['confidence_threshold']
```

Comment:
What if confidence_threshold is not provided as a config argument? 

As a best practice, the code should assume reasonable default values.

Since FallDetector extends TFImageDetector, the confidence_threshold variable is already instantiated after the `super().__init__(model, **kwargs)` call. So there is no need to declare another instance variable with the same use.

However we can move some of the code from ImageDetection that assumes that the model returns a box down to ObjectDetection. That would keep ImageDetector cleaner and more relevant to FallDetector. 

With that, we can also move the default values for confidence away from ImageDetector, since its too abstract and early to assume 0.6 is a good default. Instead the default value could go down to ObjectDetector and FallDetector respectively where we have a better idea what confidence defaults work well.

Reply:
@bhavikapanara did you see this comment about setting the `confidence_threshold` config parameter value.

Reply:
@ivelin I didn't get exactly here what I need to do?

Reply:
The essence of the issue is that your code will bomb if `confidence_threshold` is not provided explicitly in the config file. Think about setting a good default value if no explicit value is provided by the user.

Reply:
@bhavikapanara Looks like this is the last open comment for this PR. All others have been resolved.